### ISSUE TITLE
Docker_Run.Bash Example

### ISSUE THREAD

**Issue by danazc77** [EricJMarti/inventory-hunter#33]
I have completed all the steps and set everything up and confirmed I am getting the email when testing through a command "sendmail".  2 quick questions and I should be up and running.  Any help would be greatly appreciated.

1.  I tested out gmail set up and I do get an email from "sendmail" command, but when I try to restart postfix, it tells me that "sudo: systemct1: command not found.  So email is working, but not sure Postfix is actually running behind the scenes. Thoughts?

---------------------------------------------------------------------------------------------------------------------------------

2.  I am wondering if someone can give a sample layout of the final Command.   I want to make sure I am formatting correctly.  I updated my yaml files and saved them down.  Not sure how to point to the Yaml files in the command line that are saved in a folder.  Also I did GMAIL so I assume to use 127.0.0.1 for Relay ip.

$ ./docker_run.bash -c <config_file> -e <email_address> -r <relay_ip_address>

This is awesome and thank you everyone for helping me as I am 'newer" to this coding but learning.  Thank you for all you have done also Eric!

**Comment by Instah** [comment]
For me, I found that using the example given was giving me issues when looking for the files. Try to give it a detailed path for both the docker and the config. Here’s what I use:

$ ./inventory-hunter/docker_run.bash -c ./inventory-hunter/config/<file name>.yaml -e <email> -r 127.0.0.1 

(I’m not sure if the relay is correct but from personal experience, it works with gmail)

**Comment by danazc77** [comment]
Ok I got docker_run.bash to work so thank you.  What is the command to kill the container?  Sorry I am pretty new and learning a ton.  

I am glad to see the container working, but for some reason postfix is not running in the background.  I am being told systemct1 command not found.  I don't recall anything being called a systemct1 in any of these steps unless I am missing something.

**Comment by Instah** [comment]
I’m fairly new to this, I’m learning as I go haha so I’m just sharing what I know.

To stop the container use:
$ docker stop (container name)

And to delete the container use:
$ docker rm (container name)

You won’t be able to delete unless you stop it first.

And for postfix what I did was I just followed the tutorial in the README file and then used the command. 
(Make sure you have all the requirements)

**Comment by danazc77** [comment]
@Instah Thank you.. This is very helpful. I did follow the complete step by step process of the SMTP Tutorial and the email works as I can do a manual 'sendmail' command and I get the notification. It is like the job isn't set up or able to run behind the scenes.  The email doesn't come to me, but I have to initiate the email through the command line.  I will have to run through the tutorial again as I have a few times now, so I have to be missing something unless it didn't get set up with all the pieces when it downloaded.

**Comment by danazc77** [comment]
Should the Postfix be installed on the main directory or in the cd Inventory-hunter folder.. Maybe that is where it is having the disconnect.

**Comment by Instah** [comment]
I don’t know where my postfix is saved to be completely honest with you. But it’s not in the inventory-hunter folder though.

**Comment by branchslayer** [comment]
Hey everyone, new here and attempting to take a stab at this, new to linux and docker, I dont have a raspberry Pi so I used my main PC with Docker for windows. I have it downloaded and installed, I was able to run the Git command to build the Image which it did. I wrote a config for PS5, however I am unsure how to start the container using that Bash command in windows. Could someone help me out?

C:\Users\myusername\inventory-hunter>docker run docker_run.bash
Unable to find image 'docker_run.bash:latest' locally
docker: Error response from daemon: pull access denied for docker_run.bash, repository does not exist or may require 'docker login': denied: requested access to the resource is denied.
See 'docker run --help'.
